cmf-cli/resources/template_feed/init/EnvironmentConfigs/GlobalVariables.yml

```diff
@@ -9,10 +9,14 @@ variables:
   CmfCliRepository: '<%= $CLI_PARAM_CmfCliRepository %>'
   # cmf cli Version
   CmfCliVersion: '3.x.x'
+  # cmf pipeline Path
+  CmfCliPath: '$(Agent.TempDirectory)/node_modules/.bin/cmf-cli'
```

Comment:
I'm not sure this path is the same for all targets. In cloud agents, I believe this installs a level up. Check the //if statements for CLI execution in the old pipelines.